x-pack/test/accessibility/apps/ingest_node_pipelines.ts

```diff
@@ -50,8 +49,7 @@ export default function ({ getService, getPageObjects }: any) {
     });
 
     it('Create Pipeline Wizard', async () => {
-      await testSubjects.click('emptyStateCreatePipelineDropdown');
-      await testSubjects.click('emptyStateCreatePipelineButton');
+      await common.navigateToUrl('ingestPipelines', 'create', { shouldUseHashForSubUrl: false });
```

Comment:
The previous code was failing bc it was expecting the create button from the empty state view. Rather than depending on pipelines to exist/not exist, we navigate directly to the create page.